cmd/kache-doc-gen/main.go

```diff
@@ -29,12 +29,13 @@ import (
 
 	"github.com/spf13/cobra/doc"
 
+	"github.com/kasvith/kache/internal/cobra-cmds"
 	"github.com/kasvith/kache/internal/cobra-cmds/kache"
 )
 
 func main() {
 	// import all commands
-	kache.RootCmd.AddCommand(kache.VersionCmd)
+	kache.RootCmd.AddCommand(cobracmds.VersionCmd)
```

Comment:
this is a utility so i hope to exclude this from build, please revert to original :)

Reply:
remove this ? or move `kache.VersionCmd` to `cobracmds.VersionCmd`

or 

change `VersionCmd` to 
```go
var VersionCmd = &cobra.Command{
	Use:   "version",
	Short: "Display application version",
	Long:  `Display application version on the screen`,
	Run: func(cmd *cobra.Command, args []string) {
		if BuildTime == "" || GitHash == "" {
			fmt.Printf("Version: %s\n", APPVER)
			return
		}
		fmt.Printf("Version: %s, BuildTime: %s, GitHash: %s\n", APPVER, BuildTime, GitHash)
	},
}
```

Reply:
just revert the file to original

Reply:
but i move `github.com/kasvith/kache/internal/cobra-cmds/kache. VersionCmd` to `github.com/kasvith/kache/internal/cobra-cmds. VersionCmd`

to let cli reuse it.

copy a `VersionCmd` from kache to cli package?

Reply:
sorry ignore this. build script will do versioning. if in future if we want to add separate info for either of binaries we can split them to two files